conf/cassandra_latest.yaml

```diff
@@ -2359,17 +2369,33 @@ storage_compatibility_mode: NONE
 #     full:
 #       # Enable/Disable full auto-repair
 #       enabled: true
-#       # Minimum duration between repairing the same node again. This is useful for tiny clusters, such as
-#       # clusters with 5 nodes that finish repairs quickly. The default is 24 hours. This means that if the scheduler
-#       # completes one round on all nodes in less than 24 hours, it will not start a new repair round on a given node
-#       # until 24 hours have passed since the last repair.
+#       # Minimum duration between repairing the same node again. This is useful for tiny clusters,
+#       # such as clusters with 5 nodes that finish repairs quickly. This means that if the scheduler completes one
+#       # round on all nodes in less than this duration, it will not start a new repair round on a given node until
+#       # this much time has passed since the last repair completed. Consider increasing to a larger value to reduce
+#       # the impact of repairs, however note that one should attempt to run repairs at a smaller interval than
+#       # gc_grace_seconds to avoid zombies.
 #       min_repair_interval: 24h
 #       token_range_splitter:
+#         # Implementation of IAutoRepairTokenRangeSplitter; responsible for splitting token ranges
+#         # for repair assignments.
+#         #
+#         # Out of the box, Cassandra provides org.apache.cassandra.repair.autorepair.{RepairTokenRangeSplitter,
+#         # FixedTokenRangeSplitter}.
+#         #
+#         # - RepairTokenRangeSplitter (default) attempts to intelligently split ranges based on data size and partition
+#         #   count.
+#         # - FixedTokenRangeSplitter splits into fixed ranges based on the 'number_of_subranges' option.
```

Comment:
Explicitly clarify that for v nodes `number_of_subranges` will be applicable for each token range. E.g., if there are 16 v nodes, then the `number_of_subranges` will be for each v node token range.

Reply:
In auto_repair.adoc, I documented this in more detail in a way that aligns with your changes in #46, does this look alright?

I just noticed that I put 64 here, but we discussed using 32.  Based on what you ultimately go with in your PR, I can adjust this.

![image](https://github.com/user-attachments/assets/60c2beab-da6d-4927-966c-af68d5af7308)